src/app/_blocks/EpisodeHead/AudioPlayer/Buttons/index.tsx

```diff
@@ -0,0 +1,51 @@
+export function PlayButton() {
+  return (
+    <svg width="54" height="54" viewBox="0 0 54 54" fill="none" xmlns="http://www.w3.org/2000/svg">
+      <circle cx="27" cy="27" r="27" fill="white" />
+    </svg>
+  )
+}
+
+;<svg
```

Comment:
Is this svg lost?

Reply:
No, it was the preparation for the upcoming branch / custom audio player (but got caught up before branching out 😅 )